components/director/pkg/cert/certutils_test.go

```diff
@@ -99,13 +102,23 @@ func TestSubjectExtraction(t *testing.T) {
 			assert.Equal(t, testCase.orgUnit, cert.GetOrganizationalUnit(testCase.subject))
 			assert.Equal(t, testCase.orgUnits, cert.GetAllOrganizationalUnits(testCase.subject))
 			assert.Equal(t, testCase.uuidOrgUnit, cert.GetUUIDOrganizationalUnit(testCase.subject))
-			assert.Equal(t, testCase.remainingOrgUnit, cert.GetRemainingOrganizationalUnit(testCase.orgUnitPattern)(testCase.subject))
+			assert.Equal(t, testCase.remainingOrgUnit, cert.GetRemainingOrganizationalUnit(testCase.orgUnitPattern, testCase.orgRegionPattern)(testCase.subject))
 			assert.Equal(t, testCase.commonName, cert.GetCommonName(testCase.subject))
-			assert.Equal(t, testCase.possibleOrgUnitMatches, cert.GetPossibleRegexTopLevelMatches(testCase.orgUnitPattern))
+			assert.Equal(t, testCase.possibleOrgUnitMatches, cert.GetPossibleRegexTopLevelMatches(constructOURegex(testCase.orgUnitPattern, testCase.orgRegionPattern)))
 		})
 	}
 }
 
+func constructOURegex(patterns ...string) string {
```

Comment:
The same function is present in `cerutills.go`. Make it an exported function there and use `cert.ConstructOURegex` instead of duplicating it here.